docs/docs/Configuration/configuration-global-variables.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Global Variables
-sidebar_position: 0
-slug: /settings-global-variables
+sidebar_position: 5
+slug: /configuration-global-variables
```

Comment:
I can't comment on line 1 so I'm putting it here instead. Use sentence case capitalization for the title.